### ISSUE TITLE
services without image specifications cause errors for me (and may in fact be invalid)

### ISSUE THREAD

**Issue by adf-ncgr** [legumeinfo/gcv-docker-compose#7]
Just trying to get up and running off of v2.3.0-c0 I am encountering errors like:
```
sudo docker-compose -f compose.prod.yml up -d
ERROR: The Compose file is invalid because:
Service reverse-proxy has neither an image nor a build context specified. At least one must be provided.
```
at first I suspected this might be due to the VM in which I am operating being behind on docker engine version but per the compose file spec it seems to be a problem if I'm interpreting it correctly when it says:

> image MAY be omitted from a Compose file as long as a build section is declared. Compose implementations without build support MUST fail when image is missing from the Compose file.

https://github.com/compose-spec/compose-spec/blob/master/spec.md#image

**Comment by adf-ncgr** [comment]
PS. I can try to fix this unless I'm completely off-base about what's needed

**Comment by alancleary** [comment]
> PS. I can try to fix this unless I'm completely off-base about what's needed

It's a multiple compose file configuration, i.e. the `compose.yml` file provides a base configuration that the `compose.dev.yml` and `compose.prod.yml` files extend, so you need to use two compose files at a time. It also looks like you haven't specified a profile, which is now required since both GCV and its microservices are run via the same compose files but you might want to use just one or the other. For example:
```console
docker compose -f compose.yml -f compose.prod.yml --profile services up -d
```
This is all documented in the README ;)

**Comment by adf-ncgr** [comment]
thanks for clarifying- I guess that's why they don't call it SKIMME.

**Comment by alancleary** [comment]
> should I go ahead with that and corresponding doc fix?

Yes please! This sentence about the redis_loader service is also wrong: "This is the only service that can be run with just the compose.yml file." I think just removing that sentence and updating the example as you've described should be sufficient. Thanks!

**Comment by adf-ncgr** [comment]
fixed in [bb36fc6](https://github.com/legumeinfo/gcv-docker-compose/pull/9/commits/bb36fc6fae6f042c87dcadd2aef6e92abb9edd46)
